react/features/video-menu/components/web/DemoteToVisitorDialog.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { translate } from '../../../base/i18n/functions';
+import Dialog from '../../../base/ui/components/web/Dialog';
+import AbstractDemoteToVisitorRemoteParticipantDialog from '../AbstractDemoteToVisitorRemoteParticipantDialog';
+
+/**
+ * Dialog to confirm a remote participant demote action.
+ */
+class DemoteToVisitorDialog extends AbstractDemoteToVisitorRemoteParticipantDialog {
```

Comment:
If this is extending `AbstractDemoteToVisitorDialog.ts` and if we extract the submit there to an action maybe we can convert this one to a functional component since it has only render?